src/main/steps/c100-rebuild/check-your-answers/content.ts

```diff
@@ -0,0 +1,237 @@
+  MiamExemption,
+  MiamTitle,
+  PastAndCurrentProceedings,
+  TypeOfOrder,
+  WithoutNoticeHearing,
+} from './mainUtil';
+import { hearingDetailsContents } from './util/hearingwithout.util';
+import { MiamFieldsLoader } from './util/miam.util';
+import { otherProceedingsContents } from './util/otherProceeding.util';
+import { typeOfCourtOrderContents } from './util/typeOfOrder.util';
+
+export const enContent = {
+  serviceName: 'Check your answers ',
+  section: '',
+  title: 'Check your Answers',
+  change: 'Edit',
+  topWarning: 'Your answers will be shared with the other people in this case.',
+  makingSure: 'Please review your answers before you finish your application.',
+  continue: 'Accept and continue',
+  Yes: 'Yes',
+  No: 'No ',
+  SummaryDetail: 'Download a draft of your application (PDF)',
+  SummaryDetailInnerText:
+    "<p class='govuk-body'>            If you cannot open the PDF file after downloading, download and install            <a href='https://get.adobe.com/uk/reader/' class='govuk-link' rel='external' target='_blank'>Adobe Acrobat Reader</a> to try again.          </p><p class='govuk-body'>            Please note this draft is for your records. Only the completed application will be admitted in court.          </p><a class='govuk-button ga-pageLink govuk-button--secondary' role='button' draggable='false' data-module='govuk-button' data-ga-category='check your answers' data-ga-label='download draft' download='' href='/steps/completion/summary.pdf'>Download draft application</a>",
+  StatementOfTruth: {
+    title: 'Statement of Truth',
+    heading: 'Confirm before you submit the application',
+    warning:
+      'Proceedings for contempt of court may be brought against anyone who makes, or causes to be made, a false statement verified by a statement of truth without an honest belief in its truth.',
+    inset:
+      '<p>Once you submit your application, you cannot make further changes. Select Save and come back later to save your application, or select Pay and submit your application to complete your online application.</p><p>You can download a copy of your submitted application in PDF format using the link provided.</p>',
+    check: 'I believe that the facts stated in this application are true',
+    lastPara:
+      'This confirms that the information you are submitting is true and accurate, to the best of your knowledge. It’s known as your ‘statement of truth’.',
+  },
+  sectionTitles: {
+    Miam: '5. MIAM: Mediation Information and Assessment Meeting',
+    MiamAttendance: 'MIAM attendance',
+    MiamExemption: 'MIAM exemption',
+    AdvisingCourt: "6. What you're asking the court to decide",
+    WithoutNoticeHearing: '7. Hearing details',
+    ChildernDetails: "8. Childen's details",
+    InternationalElement: '11. International elements',
+    otherProceedings: '9. Past and current proceeding',
+    additionationDetailsAboutChildern: 'Additional details about the children',
+  },
+  keys: {
```

Comment:
@DaimDN We can remove these keys if we are using the labels from respective content.ts ?